src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -77,10 +78,17 @@ private void checkTwin(Path path) {
         }
     }
 
-    private void checkDot(Path file, String relativePath) {
+    private void checkDotsInFile(Path file, String relativePath) {
         if (relativePath.contains(".")) {
             String message = "Do not use dots in file names. Use nested folders instead. Wrong fixture: " + file;
             throw new LoaderException(message);
         }
     }
+
+    private void checkDotsInFolder(Path folder) {
```

Comment:
`Path folder` should be `Optional<Path> folder` if it can be null and this is an expected behaviour?